tests/components/rest/test_notify.py

```diff
@@ -0,0 +1,52 @@
+"""The tests for the rest.notify platform."""
+from os import path
+
+from homeassistant import config as hass_config
+import homeassistant.components.notify as notify
+from homeassistant.components.rest import DOMAIN
+from homeassistant.const import SERVICE_RELOAD
+from homeassistant.setup import async_setup_component
+
+from tests.async_mock import patch
+
+
+async def test_reload_notify(hass):
+    """Verify we can reload the notify service."""
+
+    assert await async_setup_component(
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6103615)